deps/key-value-storage/src/postgres/README.md

```diff
@@ -0,0 +1,54 @@
+# PostgreSQL backend for key-value storage
+
+This module provides a PostgreSQL-backed implementation of `KeyValueStorage` for the key-value storage system. Key-value pairs are stored in a single table with the value content base64-url encoded to prevent SQL injection and ensure safe storage of arbitrary text.
+
+## Features
+
+- Stores key-value pairs in a configurable table (`key` primary key, `value` as binary)
+- Safe parameter binding via `sqlx`
+- Input validation for keys (ASCII alphanumeric, `-`, `_`)
+
+## Quick start (with Docker)
+
+Use the provided helper scripts to spin up a local PostgreSQL for development:
+
+```bash
+bash set-up.sh
+```
+
+What it does:
+- Starts a container named `policy-postgres` on host port `6432`
```

Comment:
since the store is supposed to be used for all sort of state, not just policies, it probably makes sense to just call it postgres here.